bravado/client.py

```diff
@@ -293,6 +293,9 @@ def construct_request(operation, request_options, **op_kwargs):
         'params': {},  # filled in downstream
         'headers': request_options.get('headers', {}),
     }
+    # Adds Accept header to request for msgpack response if specified
+    if _use_msgpack:
```

Comment:
Please pass this flag via ``request_options`` as done for [``also_return_response``](https://github.com/Yelp/bravado/blob/2084666dba9f3407a2dbebfb96387a60b00b4e0c/bravado/client.py#L266).